app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -103,6 +105,48 @@ class WalkthroughTopicListFragmentTest {
     )
   }
 
+  // Checks if progressBar is gets hidden on scrolling to the second row of items
```

Comment:
we can skip the comment, as the test name is sufficient to understand what test we are doing here.

Reply:
Done